.eslintrc.js

```diff
@@ -68,6 +68,7 @@ module.exports = {
         eventHandlerPropPrefix: 'on',
       },
     ],
+    'react/prefer-stateless-function': 'warn',
```

Comment:
@rosskevin If we move toward using `React.propTypes` over flow for the propTypes, we should be able to remove the class overhead 🎉  (less code after transpilation)